config.coffee

```diff
@@ -9,15 +9,13 @@ module.exports =
   APPLICATION_NAME: 'force-staging'
   NODE_ENV: 'development'
   PORT: 3004
-  PROXY_API_URL: null
   API_URL: 'http://localhost:3000'
   POSITRON_URL: 'http://localhost:3005'
   FUSION_URL: null
   REQUEST_LIMIT: 120
   REQUEST_EXPIRE_MS: 60000 # 1 minute
   APP_URL: 'http://localhost:3004'
-  MOBILE_URL: 'http://localhost:3333'
-  CANONICAL_MOBILE_URL: 'https://m.artsy.net'
+  MOBILE_URL: 'https://m.artsy.net'
```

Comment:
Can't remember why back in the day we needed these to be separately configured, but considering these are pointing to the same thing in production—it doesn't seem to be necessary to duplicate them now.